sway-fmt-v2/src/utils/helpers.rs

```diff
@@ -0,0 +1,37 @@
+use crate::Formatter;
```

Comment:
We should avoid using ambiguous file names like `helpers.rs` and instead give descriptive names as to what utility the file provides like `bracket.rs` might be better here since it should only be handling brackets. If you intend on adding more punctuation handling in this file you could call it `punctuation.rs`.